### PR TITLE
Add browser version info

### PR DESCRIPTION
**Pull request by bamurtaugh**
Fixes https://github.com/microsoft/vscode/issues/195552 with the info @jrieken provided

### CODE REVIEW

File: docs/editor/vscode-web.md [hunk 1]
```diff
@@ -175,7 +175,9 @@ Edge and Chrome today support the [File System API](https://developer.mozilla.or
 
 ### Browser support
 
-You can use VS Code for the Web in Chrome, Edge, Firefox, and Safari.
+You can use VS Code for the Web in the latest versions of Chrome, Edge, Firefox, and Safari. Older versions of each browser may not work - we only guarantee support for the latest version. 
+
+> **Tip:** A good indication of which browser version will work is the version of [Playwright](https://playwright.dev/) we're using in VS Code. The browser versions it comes with is what our tests assume exist. You can check out the VS Code repo's [`package.json`](https://github.com/microsoft/vscode/blob/main/package.json) to find the version of Playwright in `devDependencies/@playwright/test`.
```
Comment (gregvanl) [inline docs/editor/vscode-web.md:180]: How about:
"One way to check the compatible browser version is to look at the version of [Playright]((https://playwright.dev) currently used for testing VS Code and review its supported browser versions. You can find the currently used Playwright version in the VS Code repo's [package.json](https://github.com/microsoft/vscode/blob/main/package.json) file at `devDependencies/@playwright/test`. Once you know the Playwright version, for example `1.37`, you can then review the **Browser Versions** section in their [Release notes](https://playwright.dev/docs/release-notes)."

**Review by jrieken — DISMISSED** [review]
+1 for Greg's suggestion

**Comment by bamurtaugh** [comment]
Thanks both! Just updated, lmk what you think.

**Review by jrieken — APPROVED** [review]

**Comment by yiliang114** [comment]
According to your recommended query method, I made [a side project](https://cloud-webide.github.io/) and used a script to extract and aggregate the browser information supported by playwright supported by each release of vscode,  maybe there is not a lot of information extracted now, but I think it brings convenience to my query.

**Pull request merged**